src/sidebar/helpers/test/mentions-test.js

```diff
@@ -0,0 +1,122 @@
+import { renderMentionTags, unwrapMentions, wrapMentions } from '../mentions';
+
+[
+  // Mention at the end
+  {
+    text: 'Hello @sean',
+    authority: 'hypothes.is',
+    textWithTags:
+      'Hello <a data-hyp-mention="" data-userid="acct:sean@hypothes.is">@sean</a>',
```

Comment:
Might be worth extracting the tag formatting here into a helper so these tests can be easily updated if the format changes.

Reply:
Do you mean a local function in this test like ``(username, authority) => `<a data-hyp-mention="" data-userid="acct:${username}@${authority}">@${username}</a>` ``, or something more elaborated and production-ready?

Reply:
Actually, the "production-ready" function is basically this few lines of logic https://github.com/hypothesis/client/pull/6797/files#diff-7ea26cd684b673e7c47a1b00e5fce3d1dcc13f50c1fb9d8ad694eb7888101776R20-R24 😅 

But using a function when testing another function that also uses that function seems like `assert.equa(true, true)`, haha.

I'll just create a local function in the test that allows to update it in a single place, but does not couple with production code.